tools/oversight/explorer.html

```diff
@@ -1,4 +1,4 @@
-<html>
+<html></html>
```

Comment:
This looks wrong.

Reply:
fixed in [1d491f8](https://github.com/adobe/helix-website/pull/756/commits/1d491f8b96fed532343f554374e1b1d213405881)